plugins/ebitdo/fu-ebitdo-plugin.c

```diff
@@ -19,6 +19,7 @@ G_DEFINE_TYPE(FuEbitdoPlugin, fu_ebitdo_plugin, FU_TYPE_PLUGIN)
 static void
 fu_ebitdo_plugin_init(FuEbitdoPlugin *self)
 {
+	fu_plugin_add_flag(FU_PLUGIN(self), FWUPD_PLUGIN_FLAG_IMMUTABLE_ENUMERATION);
```

Comment:
I don't think this one is true

Reply:
It doesn't automatically detach to bootloader mode nor does it detach a kernel driver does it?  I think it's true for 8bitdo that it's immutable enumeration.